pandas/core/categorical.py

```diff
@@ -1340,8 +1351,14 @@ def _repr_categories_info(self):
         """ Returns a string representation of the footer."""
 
         category_strs = self._repr_categories()
-        levheader = "Categories (%d, %s): " % (len(self.categories),
-                                               self.categories.dtype)
+
+        from pandas.tseries.period import PeriodIndex
```

Comment:
yeh ``PeriodIndex`` acts slightly differently. what I would do it maybe have another property of ``Index`` , maybe ``.dtype_values`` which returns ``self.dtype``, but for ``PeriodIndex`` returns ``period``. (this will fix some other uses of ``isinstance(...., PeriodIndex)`` type of things (in other parts of the code)